docs/userguides/accounts.md

```diff
@@ -234,6 +234,27 @@ message = encode_defunct(text="Hello Apes!")
 signature = account.sign_message(message)
 ```
 
+## Signed Message
+
+Get the signed message signature as well as message hash signed by the account
```

Comment:
Can we reformat this section to show how to use the new property to recover a message signer?

Reply:
I don't think we need `messageHash` to verify a contract. Ex. usecase would be to verify contracts. As mentioned here https://eips.ethereum.org/EIPS/eip-191

Reply:
> I don't think we need `messageHash` to recover a signer. Ex. use case would be to verify contracts. As mentioned here https://eips.ethereum.org/EIPS/eip-191

did you mean to link something else?

Reply:
Nope. I wanted to show this EIP to make sure I'm understanding it correctly. here is a use case I found using web3.py https://github.com/0xMaka/w3py/blob/d4c6d945002fa3a6990dba19cd8c6ee970156dbe/permit/pysign.py

Reply:
I don't think that's an EIP

In general, most signature-based EIPs require passing individual data elements through the contract call so they can be individually verified before they are hashed together

Reply:
Do you have any examples in mind ?

Reply:
I'm still confused about the use case

Reply:
Well that's sorta my point, I don't think I know of a single production example where this happens in practice

I thought this field was for help in doing recovery off-chain

Reply:
Maybe something like this? Say we have an anonymous messaging system or something like that.
1. Alice sends a message & the system stores the message_hash. 
2. Bob sends the same message & pretends to be Alice. 
3. By verifying the message_hash we can say that Bob is doing something suspicious 

All without passing any signatures ? If it makes sense.